tools/rosgraph/src/rosgraph/xmlrpc.py

```diff
@@ -106,7 +109,7 @@ def _support_http_1_1(major, minor):
 
 
 class SilenceableXMLRPCRequestHandler(SimpleXMLRPCRequestHandler):
-    if _support_http_1_1(4,15):
+    if _support_http_1_1:
```

Comment:
this is a function - don't you need to call it?

Reply:
arg, dumb error